MusicTaste/MusicTaste/View Controllers/LoginViewController.m

```diff
@@ -28,32 +28,34 @@ - (IBAction)registerbutton:(id)sender {
     
 }
 - (void)registerUser {
-    PFUser *newUser = [PFUser user];
-    newUser.username = self.usernameField.text;
-    newUser.password = self.passwordField.text;
-    [newUser signUpInBackgroundWithBlock:^(BOOL succeeded, NSError * error) {
-        if (error != nil) {
-            NSLog(@"Error: %@", error.localizedDescription);
-        } else {
-            NSLog(@"User registered successfully");
-            
-            [self performSegueWithIdentifier:@"loginSegue" sender:nil];
-        }
-    }];
+    [self performSegueWithIdentifier:@"loginSegue" sender:nil];
+//    PFUser *newUser = [PFUser user];
+//    newUser.username = self.usernameField.text;
+//    newUser.password = self.passwordField.text;
+//    [newUser signUpInBackgroundWithBlock:^(BOOL succeeded, NSError * error) {
+//        if (error != nil) {
+//            NSLog(@"Error: %@", error.localizedDescription);
+//        } else {
+//            NSLog(@"User registered successfully");
+//
+//            [self performSegueWithIdentifier:@"loginSegue" sender:nil];
+//        }
+//    }];
```

Comment:
Try not to include commented code in Pull Requests. The are several alternatives:

* You can save such code in your notes in a local file on your computer.
* You can shelve the changes in git
* You can branch and commit the commented code in a git commit in the separate branch, separate from dev